cis_v130/docs/cis_v130_5_1.md

```diff
@@ -16,6 +16,6 @@ Public access to remote server administration ports, such as 22 and 3389, increa
 5. Choose the **Inbound Rules** tab and click **Edit Inbound rules**
 6. Check remote server access port entries. **Note** A Port value of ALL or a port range such as 0-1024 are inclusive of port 22, 3389, and other remote server administration ports.
 7. Either
-    - A) Update the `source` field to a range other than 0.0.0.0/0 or
+    - A) Update the `source` field to a range other than 0.0.0.0/0 or ::/0
```

Comment:
```suggestion
    - A) Update the `source` field to a range other than 0.0.0.0/0 or
```

@akumar-99 I agree that this change is more helpful, but in this case, I don't believe we'll deviate from the official CIS documentation. Similar to my rationale in https://github.com/turbot/steampipe-mod-aws-compliance/issues/509#issuecomment-1263828637, it's often easier for us to stay with CIS and reduces possible confusion between our documentation and their's.

In terms of seeing this change in future versions, one suggestion I have is to submit a change or open a discussion on the CIS WorkBench website for the upcoming AWS v1.6.0 benchmark and propose this change.

Reply:
@cbruno10 I have a favour to ask from you. Can you please help me with the procedure to propose a change on CIS WorkBench?

Reply:
@akumar-99 I'll see if we can propose additional changes in the next few weeks. Proposing changes on the site isn't difficult, but time consuming, so it's often a lower priority for us (among all other plugin + mod work).

Reply:
Got it. Thank you :)